mail/postfix/src/opnsense/mvc/app/controllers/OPNsense/Postfix/forms/general.xml

```diff
@@ -35,6 +35,12 @@
         <type>text</type>
         <help>Port to listen on.</help>
     </field>
+    <field>
+        <id>general.inet_protocol</id>
```

Comment:
in these cases I have to insist on changing the internal name as well especially since it is still new, sorry :)

Reply:
No problem, in the end it's the same result :)